apps/system/test/marionette/browser_video_test.js

```diff
@@ -76,6 +76,7 @@ marionette('Video', function() {
       return player.isFullScreen();
     });
 
+    client.helper.wait (1000);
```

Comment:
Let's add a comment here explaining what we are waiting for. It sounds like we are essentially waiting to make sure something *doesn't* happen, which can only be achieved with a sleep like this. But if that's the case, let's make that explicit in a comment.